aws/services/waf/resource.ftl

```diff
@@ -614,12 +637,6 @@
                         content=[
                             r' case ${STACK_OPERATION} in',
                             r'    create|update|delete)',
-                            r'       manage_waf_logging ' +
-                            r'          "' + getRegion() + r'"' +
-                            r'          "' + wafaclId + r'"' +
-                            r'          "' + wafType + r'"' +
-                            r'          "disable"' +
-                            r'          || return $?',
```

Comment:
Should this be removed from V1? this PR said it was for v2 only?

Reply:
I can't find nor remember why I removed that so I reinstated it because I can only think it was an error